util/cpeutils.c

```diff
@@ -801,23 +803,30 @@ get_fs_component (const char *fs_cpe, int index)
         {
           if (*c == ':' && c == fs_cpe)
             i++;
```

Comment:
Sorry, but I don't see that point. In the lines listed above, *c can not be an escaped colon because c == fs_cpe and we are at the beginning of the CPE string. In all other cases "escaped" is set back to false after each following character I suppose. What do you think?

Reply:
But nevertheless, the if clause seems to be obsolete now.

Reply:
Oh right, it was at the beginning of the string, so it was safe.